# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1447262530 18000
#      Wed Nov 11 12:22:10 2015 -0500
# Node ID 0dfeb5c82750d85871aaaf1c33f45646ba695ff7
# Parent  6faaab833605973694a8ce545df8e84544a11456
* ov.h (OV_REP_TYPE): Delete obsolete macro.

diff --git a/libinterp/octave-value/ov.h b/libinterp/octave-value/ov.h
--- a/libinterp/octave-value/ov.h
+++ b/libinterp/octave-value/ov.h
@@ -1377,22 +1377,16 @@ OV_BINOP_FN (op_struct_ref)
 
 OV_COMP_BINOP_FN (op_trans_mul)
 OV_COMP_BINOP_FN (op_mul_trans)
 OV_COMP_BINOP_FN (op_herm_mul)
 OV_COMP_BINOP_FN (op_mul_herm)
 
 extern OCTINTERP_API void install_types (void);
 
-// This will eventually go away, but for now it can be used to
-// simplify the transition to the new octave_value class hierarchy,
-// which uses octave_base_value instead of octave_value for the type
-// of octave_value::rep.
-#define OV_REP_TYPE octave_base_value
-
 // Templated value extractors.
 template<class Value>
 inline Value octave_value_extract (const octave_value&)
 { assert (false); }
 
 #define DEF_VALUE_EXTRACTOR(VALUE,MPREFIX) \
 template<> \
 inline VALUE octave_value_extract<VALUE> (const octave_value& v) \
